web/server/config/i18n/sync/copyOnce.ts

```diff
@@ -0,0 +1,9 @@
+import glob from 'glob';
```

Comment:
This is probably the culprit...

Maybe we can move this code to the gen:games script ?